src/PKSim.Core/Model/TransportDirection.cs

```diff
@@ -108,7 +108,7 @@ public static TransportDirectionId DefaultBloodCellsDirectionFor(TransportType t
             case TransportType.Influx:
                return InfluxPlasmaToBloodCells;
             case TransportType.BiDirectional:
-               return BiDirectionalPlasmaBloodCells;
+               return BiDirectionalBloodCellsPlasma;
```

Comment:
We already have expression in plasma so in order for the transport to be allocated, I needed to change the src. So the bidirectional between Blood cells and Plasma will be located in BloodCells